File: JS/ejercicio60.js
let votos = [
    "Ana",
    "Juan",
    "Ana",
    "Pedro",
    "Ana",
    "Juan"
];

let ana = 0;
let juan = 0;
let pedro = 0;

for (let i = 0; i < votos.length; i++) {
    if (votos[i] == "Ana") {
        ana++;
    } else if (votos[i] == "Juan") {
        juan++;
    } else if (votos[i] == "Pedro") {
        pedro++;
    }
}

console.log("Ana:", ana);
console.log("Juan:", juan);
console.log("Pedro:", pedro);

if (ana > juan && ana > pedro) {
    console.log("Ganador: Ana");
} else if (juan > ana && juan > pedro) {
    console.log("Ganador: Juan");
} else {
    console.log("Ganador: Pedro");
}

File: PYTHON/ejercicio60.py
ana = 0
juan = 0
pedro = 0

cantidad = int(input("Cantidad de votos: "))

for i in range(cantidad):
    voto = input("Vote por Ana, Juan o Pedro: ")

    if voto == "Ana":
        ana += 1
    elif voto == "Juan":
        juan += 1
    elif voto == "Pedro":
        pedro += 1

print("Ana:", ana)
print("Juan:", juan)
print("Pedro:", pedro)

if ana > juan and ana > pedro:
    print("Ganador: Ana")
elif juan > ana and juan > pedro:
    print("Ganador: Juan")
else:
    print("Ganador: Pedro")